src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -2543,8 +2719,8 @@ namespace ts.projectSystem {
                 options: {}
             });
             projectService.checkNumberOfProjects({ configuredProjects: 2 });
-            checkProjectActualFiles(projectService.configuredProjects[0], [cLib.path, cTsconfig.path]);
-            checkProjectActualFiles(projectService.configuredProjects[1], [dLib.path, dTsconfig.path]);
+            checkProjectActualFiles(configuredProjectAt(projectService, 0), [cLib.path, cTsconfig.path]);

```

Comment:
I would leave this as is as its too much work just to go look for all the instances of configured project query and number of projects checking with not much gain.